notebooks/215-image-inpainting/215-image-inpainting.ipynb

```diff
@@ -0,0 +1,350 @@
+    "\n",
+    "Note that both input shapes are the same however, the second input has a channel of 1 (monotone)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2606632b-41a3-41e4-88ae-935029ec87df",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "N, H, W, C = input_layer.shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1b641584-e018-49ce-a7df-f48d1eae1d52",
+   "metadata": {},
+   "source": [
+    "### Create a square mask\n",
+    "\n",
+    "Next, we will create a single channeled mask that will be laid on top of the original image."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "885c49ff-34bc-476e-b47b-9dd37e4edc50",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_mask(image_width, image_height, size_x=30, size_y=30, number=1):\n",
+    "    \"\"\"\n",
+    "    Create a square mask of defined size on a random location\n",
+    "\n",
+    "    :param: image_width: width of the image\n",
+    "    :param: image_height: height of the image\n",
+    "    :param: size: size in pixels of one side\n",
+    "    :returns:\n",
+    "            mask: grayscale float32 mask of size shaped [image_height, image_width, 1]\n",
+    "    \"\"\"\n",
+    "\n",
+    "    mask = np.zeros((image_height, image_width, 1), dtype=np.float32)\n",
+    "    for _ in range(number):\n",
+    "        start_x = np.random.randint(image_width - size_x)\n",
+    "        start_y = np.random.randint(image_height - size_y)\n",
+    "        cv2.rectangle(img=mask,\n",
+    "                      pt1=(start_x, start_y),\n",
+    "                      pt2=(start_x + size_x, start_y + size_y),\n",
+    "                      color=(1, 1, 1),\n",
+    "                      thickness=cv2.FILLED)\n",
+    "    return mask"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d54d4d89-e7e4-46f7-b58f-858c2f8474c8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Generate a square mask of size WxH with number of \"holes\"\n",
+    "mask = create_mask(image_width=W, image_height=H, size_x=50, size_y=50, number=15)\n",
+    "# This mask will be laid over the input image as noise\n",
+    "plt.figure(figsize=(16, 12))\n",
+    "plt.imshow(cv2.cvtColor(mask, cv2.COLOR_BGR2RGB));"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2561cfd2-f1ac-4121-99ad-019c174a10a8",
+   "metadata": {},
+   "source": [
+    "### Load and Resize the Image\n",
+    "\n",
+    "This image will be altered using the mask."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "89ce844f-35c0-4594-a4d4-c8fb1946c7f2",
+   "metadata": {
+    "pycharm": {
+     "is_executing": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Download an image\n",
+    "url = \"https://user-images.githubusercontent.com/4547501/165946837-007d9f76-5418-41f3-92c9-2f37091b72dd.png\"\n",
```

Comment:
```suggestion
    "url = \"https://www.intel.com/content/dam/www/central-libraries/us/en/images/arc-home-hero-128.png.rendition.intel.web.480.360.png\"\n",
```
Please change to something Intel related. If you don't like this image, please suggest an alternative